subprojects/core/src/main/java/org/gradle/internal/cleanup/BuildOperationBuildOutputDeleterDecorator.java

```diff
@@ -45,7 +46,8 @@ public void run(BuildOperationContext context) {
 
             @Override
             public BuildOperationDescriptor.Builder description() {
-                return BuildOperationDescriptor.displayName("Clean stale outputs for " + gradle.getIdentityPath().getName());
+                String rootProjectIdentifier = GUtil.elvis(gradle.getIdentityPath().getName(), "root build");
```

Comment:
Should we reuse the logic from here? https://github.com/gradle/gradle/blob/1c585c1e70481cd40253381fa982b269e3d31578/subprojects/core/src/main/java/org/gradle/caching/internal/BuildCacheServiceProvider.java#L197-L202